lib/server.js

```diff
@@ -92,6 +92,14 @@ Server.prototype.start = function () {
   this._injector.invoke(this._start, this)
 }
 
+/**
+ * Backward-compatibility with karma-intellij bundled with WebStorm
+ * @deprecated since version 0.13, to be removed in 0.14
+ */
+Server.start = function (config) {
```

Comment:
This will not get picked up, because of https://github.com/karma-runner/karma/blob/master/lib/index.js#L9-L18